src/styles/index.scss

```
@@ -1,3 +1,3 @@
-body {
-  margin: 0;
-}
+@import './veriables';
```

Rename file and update import
```suggestion
@import './variables';
```

---

Not fixed